src/utils/executeELM.js

```diff
@@ -103,7 +103,14 @@ function doSearch(client, release, type, collector) {
   const resources = [];
   const uri = `${type}?${params}`;
   return new Promise((resolve) => {
-    const results = client.patient.request(uri, {
+    const results = client.patient.request({
+        url: uri,
+        headers: {
+          'Cache-Control': 'no-cache, no-store, max-age=0',
+          'Expires': 'Tue, 01 Jan 1980 1:00:00 GMT',
+          'Pragma': 'no-cache'
```

Comment:
Did you find more than `Cache-Control': 'no-cache'` was needed? Are these other headers necessary, and what do they do?

Reply:
I guess my concern is that not all browsers treat the HTTP directives the same:  I see a discussion here:  https://stackoverflow.com/questions/49547/how-do-we-control-web-page-caching-across-all-browsers

Reply:
If I understand what's going on correctly, HAPI is caching results, so this shouldn't be affected by browsers

Reply:
Thanks for the change!